drivers/telescope/lx200_10micron.cpp

```diff
@@ -1274,27 +1240,51 @@ bool LX200_10MICRON::ISNewText(const char *dev, const char *name, char *texts[],
             LOGF_INFO("Model saved with name %s", NewModelNameT[0].text);
             return true;
         }
-        if (strcmp(name, TLE_TEXT) == 0)
+        if (strcmp(name, "SAT_TLE_TEXT") == 0)
         {
-          IUUpdateText(&TLEtoUploadTP, texts, names, n);
-          if (0 == SetTLEtoFollow(TLEtoUploadT[0].text))
+
+          IUUpdateText(&TLEtoTrackTP, texts, names, n);
+          if (0 == SetTLEtoFollow(TLEtoTrackT[0].text))
             {
-              TLEtoUploadTP.s = IPS_OK;
+              TLEtoTrackTP.s = IPS_OK;
               TLEfromDatabaseNP.s = IPS_IDLE;
-              IDSetText(&TLEtoUploadTP, nullptr);
+              IDSetText(&TLEtoTrackTP, nullptr);
               IDSetNumber(&TLEfromDatabaseNP, nullptr);
-              LOGF_INFO("Selected TLE %s", TLEtoUploadT[0].text);
+              LOGF_INFO("Selected TLE %s", TLEtoTrackT[0].text);
               return true;
             }
           else
             {
-              TLEtoUploadTP.s = IPS_ALERT;
+              TLEtoTrackTP.s = IPS_ALERT;
               TLEfromDatabaseNP.s = IPS_IDLE;
-              IDSetText(&TLEtoUploadTP, nullptr);
+              IDSetText(&TLEtoTrackTP, nullptr);
               IDSetNumber(&TLEfromDatabaseNP, nullptr);
               LOG_ERROR("TLE was not correctly uploaded");
               return false;
             }
+
+           return true;
+        }
+        if (strcmp(name, "SAT_PASS_WINDOW") == 0)
+        {
+
+          IUUpdateText(&SatPassWindowTP, texts, names, n);
+          if (0 == CalculateSatTrajectory(SatPassWindowT[0].text, SatPassWindowT[1].text))
+            {
```

Comment:
spacing

Reply:
Which spacing? If its the one before `IUUPdateText`, just fixed it :+1: